pages/_app.tsx

```diff
@@ -67,7 +67,7 @@ const InnerApp: FC<AppProps> = ({ Component, pageProps }: AppPropsWithLayout) =>
 					<Layout>
 						<SystemStatus>{getLayout(<Component {...pageProps} />)}</SystemStatus>
 					</Layout>
-					<ReactQueryDevtools />
+					<ReactQueryDevtools position="top-left" />
```

Comment:
Do we want to leave this here permanently? I think that this is better for mobile, but is it okay on desktop?